secure_message/resources/messages.py

```diff
@@ -114,17 +114,17 @@ def _alert_listeners(message):
         try:
             MessageSend._try_send_alert_email(message)
             MessageSend._inform_case_service(message)
-        except Exception as e:  # NOQA
+        except Exception as e:  # NOQA pylint:disable=broad-except
```

Comment:
Won't affect the mergability of this PR, but do you know if this super broad exception is legit or just laziness with exception typing?

Reply:
The try send alert message sends via gov.notify and case service via a post. The idea here was that no matter what went wrong then informing other services of the new message should not cause an error on the message save.Hence the catch all .